bitnami/apisix/templates/dashboard/pdb.yaml

```diff
@@ -3,30 +3,30 @@ Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.dataPlane.pdb.create }}
+{{- if .Values.dashboard.pdb.create }}
 apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
 kind: PodDisruptionBudget
 metadata:
-  name: {{ template "apisix.data-plane.fullname" . }}
+  name: {{ template "apisix.dashboard.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
```

Comment:
You have not adapted to the new helper's format here.

Reply:
Good catch! Done at https://github.com/bitnami/charts/pull/18456/commits/1fed0db0cec55398dcb476839ef72ba090a30091